### ISSUE TITLE
Wrong definition of the skos:hasTopConept property?

### ISSUE THREAD

**Issue by idimopoulos** [SEMICeu/ADMS-AP#2]
From adms definition:
```
<rdf:Property rdf:about="http://www.w3.org/2004/02/skos/core#hasTopConcept">
    <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#DatatypeProperty"/>
    <dc:identifier xml:lang="en">skos:hasTopConcept</dc:identifier>
    <rdfs:comment xml:lang="en">Concept that is the top level of the Theme Taxonomy.</rdfs:comment>
    <rdfs:isDefinedBy rdf:resource="http://www.w3.org/2004/02/skos/core#"/>
    <rdfs:label xml:lang="en">Theme Taxonomy - has Top Concept</rdfs:label>
    <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Literal"/>
    <rdfs:domain rdf:resource="http://www.w3.org/2004/02/skos/core#Concept"/>
    <ns1:usageNote xml:lang="en">Recommended Property - 0..n</ns1:usageNote>
  </rdf:Property>
```

In the above property, the range is declared as literal in adms.
Also, domain is declared as a dcat:Concept.
However, according to https://www.w3.org/TR/skos-reference/#concepts and https://www.w3.org/TR/skos-reference/#schemes the hasTopConcept is a property of dcat:ConceptScheme and the range of the property is of dcat:Concept.

Other sources:
https://www.w3.org/TR/skos-reference/#hasTopConcept:
The range of the hasTopConcept is a Concept, which is an object.

Am I missing something? It seems to me as a wrong definition, sorry if I am wrong.

**Comment by makxdekkers** [comment]
@idimopoulos You are absolutely right. This is a bug that needs to be corrected. In the specification, the domain is correctly stated as skos:ConceptScheme but with the incorrect range of Literal.

**Comment by akuckartz** [comment]
:+1:

How can such errors be avoided in the future?

**Comment by makxdekkers** [comment]
A sensible way to avoid such errors would be not to include domain and range axioms for properties defined elsewhere.

**Comment by EmidioStani** [comment]
Using Protege it is possible to import axioms (classes, properties) from other ontologies (opened in the same windows) which could have avoided the problem, see:

https://protegewiki.stanford.edu/wiki/P4MoveAxioms

**Comment by makxdekkers** [comment]
Issue to be resolved in the next major semantic release as there are implications for interoperability if  there are implementations that use the incorrect range.

**Comment by claudiu-cristea** [comment]
@makxdekkers is this resolved, or is there a PR for this?

**Comment by makxdekkers** [comment]
@claudiu-cristea I do not know ; I haven't been involved in this work for over a year. Probably the people who are now responsible for the maintenance of ADMS can answer your questions.

**Comment by addragan** [comment]
@claudiu-cristea we are discussing having a release on ADMS this autumn. We will get back with a detailed planning to the Working Group and evaluate and prioritise all open issues. Meanwhile, please contribute with your comments or new requests in the ADMS repository and they will be taken into account for the next release.

**Comment by claudiu-cristea** [comment]
@addragan thank you.
